### ISSUE TITLE
Documentation build broken

### ISSUE THREAD

**Issue by mouse07410** [randombit/botan#935]
```
. . . . .
/Users/mouse/src/botan/doc/manual/index.rst:15: WARNING: download file not readable: /Users/mouse/src/botan/latex/botan.pdf
. . . . .
```

The above breaks Travis CI.

**Comment by securitykernel** [comment]
Somehow 23c30819 didn't land on master, sorry. Fix is in #937.

**Comment by randombit** [comment]
Fixed now, thanks!
